docs/modules/indexes/vectorstores/examples/redis.ipynb

```diff
@@ -147,6 +138,79 @@
     "results = rds.similarity_search(query)\n",
     "print(results[0].page_content)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## RedisVectorStoreRetriever\n",
+    "\n",
+    "Here we go over different options for using the vector store as a retriever.\n",
+    "\n",
+    "There are three different search methods we can use to do retrieval. By default, it will use semantic similarity."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "retriever = rds.as_retriever()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "docs = retriever.get_relevant_documents(query)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also use similarity_limit as a search method. This is only return documents if they are similar enough"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "retriever = rds.as_retriever(search_type=\"similarity_limit\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Here we can see it doesn't return any results because there are no relevant documents\n",
+    "retriever.get_relevant_documents(\"where did ankush go to college?\")"
```

Comment:
Maybe manually add the threshold here so people can see that it's a tunable value?

Reply:
nice i like it